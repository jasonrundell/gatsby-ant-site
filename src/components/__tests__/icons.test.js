import React from 'react'
import renderer from 'react-test-renderer'

import Icon from '../icons'

describe('Icon', () => {
  it('checkmark renders correctly', () => {
    const tree = renderer.create(<Icon icon="checkmark" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('dot renders correctly', () => {
    const tree = renderer.create(<Icon icon="dot" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('tooltip renders correctly', () => {
    const tree = renderer.create(<Icon icon="tooltip" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
