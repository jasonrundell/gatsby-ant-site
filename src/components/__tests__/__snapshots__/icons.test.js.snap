// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon checkmark renders correctly 1`] = `
<span
  className="checkmark"
>
  <span>
    checkmark
  </span>
</span>
`;

exports[`Icon dot renders correctly 1`] = `
<span
  className="dot"
>
  <span>
    dot
  </span>
</span>
`;

exports[`Icon tooltip renders correctly 1`] = `
<span
  className="tooltip"
>
  <span>
    tooltip
  </span>
</span>
`;
